src/code42cli/file_readers.py

```diff
@@ -17,31 +19,44 @@ def read_csv_arg(headers):
 
 
 def read_csv(file, headers):
```

Comment:
This method is getting massive and could be more readable through the use of helper methods

Reply:
yeah, good call. I'll split it out so it reads cleaner

Reply:
I struggled to find anything to pull out into a function that really helped readability, but did shorten it overall a bit by using a comprehension for filtering extra rows.